apps/dashboard/src/components/workflow-editor/steps/email/blocks/headers.tsx

```diff
@@ -0,0 +1,207 @@
+                  height: '32',
+                  alignment: 'left',
+                  externalLink: null,
+                  isExternalLinkVariable: false,
+                  isSrcVariable: false,
+                  showIfKey: null,
+                },
+              },
+            ],
+          },
+          {
+            type: 'column',
+            attrs: {
+              columnId: '6feb593e-374a-4479-a1c7-872c60c2f4e0',
+              width: 'auto',
+              verticalAlign: 'bottom',
+            },
+            content: [
+              {
+                type: 'heading',
+                attrs: {
+                  textAlign: 'right',
+                  level: 3,
+                  showIfKey: null,
+                },
+                content: [
+                  {
+                    type: 'text',
+                    marks: [{ type: 'bold' }],
+                    text: 'Weekly Newsletter',
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      })
+      .run();
+  },
+};
+
+export const headerLogoWithText: BlockItem = {
+  title: 'Logo with Text',
+  description: 'Header with logo & text',
+  searchTerms: ['logo', 'text'],
+  icon: <EmailHeaderLogoWithText className="size-4" />,
+  command: ({ editor, range }) => {
+    editor
+      .chain()
+      .deleteRange(range)
+      .insertContent([
+        {
+          type: 'image',
+          attrs: {
+            src: '/images/email-editor/header-logo.png',
+            alt: null,
+            title: null,
+            width: '48',
+            height: '48',
+            alignment: 'center',
+            externalLink: null,
+            isExternalLinkVariable: false,
+            isSrcVariable: false,
+            showIfKey: null,
+          },
+        },
+        { type: 'spacer', attrs: { height: 8, showIfKey: null } },
+        {
+          type: 'heading',
+          attrs: { textAlign: 'center', level: 3, showIfKey: null },
+          content: [{ type: 'text', text: 'Company' }],
+        },
+      ])
+      .run();
+  },
+};
+
+export const headerLogoWithCoverImage: BlockItem = {
+  title: 'Logo with cover image',
+  description: 'Header with logo & cover image',
+  searchTerms: ['logo', 'cover', 'image'],
+  icon: <EmailHeaderLogoWithCoverImage className="size-4" />,
+  command: ({ editor, range }) => {
+    const todayFormatted = new Date().toLocaleDateString('en-US', {
+      year: 'numeric',
+      month: 'short',
+      day: 'numeric',
+    });
+
+    editor
+      .chain()
+      .deleteRange(range)
+      .insertContent({
+        type: 'section',
+        attrs: { showIfKey: null, backgroundColor: '#FFFFFF', borderWidth: 0 },
+        content: [
+          {
+            type: 'image',
+            attrs: {
+              src: '/images/email-editor/header-hero-image.png',
+              alt: null,
+              title: null,
+              width: 600,
+              height: 314,
```

Comment:
this `314` causes the empty space under the image for some reason, if I removed it, it works correctly:

![image](https://github.com/user-attachments/assets/0eb2b2ee-9136-4c13-b8f1-9eef40ef94aa)